sdk/tcp_listener.go

```diff
@@ -14,6 +16,14 @@ const (
 )
 
 func newTCPListener(address string, pluginName string) (net.Listener, string, error) {
+	if address == "" {
+		return nil, "", fmt.Errorf("plugin address cannot be empty")
+	}
+	addrParts := strings.SplitN(address, ":", 2)
```

Comment:
Does this need to take IPv6 into account, and use [` SplitHostPort`](https://golang.org/pkg/net/#SplitHostPort)?